device_doctor/lib/src/ios_device.dart

```diff
@@ -47,6 +48,15 @@ class IosDeviceDiscovery implements DeviceDiscovery {
     return results;
   }
 
+  /// iOS device property check is not needed at this moment.
+  ///
+  /// But we will implement this to replace existing chromium side bot config logic
+  /// after devicelab migration to LUCI is done.
+  @override
+  Future<Map<String, List<String>>> checkDeviceProperties({ProcessManager processManager}) async {
```

Comment:
deviceProperties only? this is actually just getting the properties without running any checks on them.

Reply:
Updated.